### PR TITLE
Add Prometheus exporter plugin for OpenSearch

### PR DESCRIPTION
**Pull request by lukas-vlcek**
### Description
Add Prometheus exporter plugin for OpenSearch.
 
### Issues Resolved
[Forum discussion](https://discuss.opendistrocommunity.dev/t/prometheus-exporter-plugin-1-3-0-0-released/8980/2)

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

Signed-off-by: Lukáš Vlček <lukas.vlcek@aiven.io>

### CODE REVIEW

File: _community_projects/prometheus-exporter-plugin.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: Prometheus exporter plugin for OpenSearch
+description: The Prometheus® exporter plugin for OpenSearch® exposes many OpenSearch metrics in Prometheus format.
+owner: Aiven.io and project maintainers
+link: https://github.com/aiven/prometheus-exporter-plugin-for-opensearch
+download_link: https://github.com/aiven/prometheus-exporter-plugin-for-opensearch/releases
+license: Apache License 2.0
+license_link: https://www.apache.org/licenses/LICENSE-2.0
+# Prometheus logo source: https://www.linuxfoundation.org/prometheus-2/ 
+icon: '/assets/media/community_projects/prometheus.svg'
```
[the remark this answers is not in the document]
Comment (stockholmux) [reply _community_projects/prometheus-exporter-plugin.md:10]: You can also remove the SVG file.

**Review by stockholmux — CHANGES_REQUESTED** [review]
Awesome contribution (more so the exporter than this YML :) )

A couple of small changes.

File: _community_projects/prometheus-exporter-plugin.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lukas-vlcek) [reply _community_projects/prometheus-exporter-plugin.md]: Done.

File: _community_projects/prometheus-exporter-plugin.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+name: Prometheus exporter plugin for OpenSearch
+description: The Prometheus® exporter plugin for OpenSearch® exposes many OpenSearch metrics in Prometheus format.
```
[the remark this answers is not in the document]
Comment (lukas-vlcek) [reply _community_projects/prometheus-exporter-plugin.md]: Done.

**Comment by lukas-vlcek** [comment]
PR updated. Thanks!

**Comment by lukas-vlcek** [comment]
Thanks @stockholmux

**Pull request merged**